README.md

```diff
@@ -137,7 +137,11 @@ This section describes how to import the **Netscout TestStream L1 Shell** and co
   2. Extract the *Netscout TestStream shell zip* package to the following location on the Quali Server machine: 
   *C:\Program Files (x86)\QualiSystems\CloudShell\Server\Drivers*
   
-  3. Run the *C:\Program Files (x86)\QualiSystems\CloudShell\Server\Drivers\cloudshell-L1-netscout_teststream\install_driver.bat* file.
```

Comment:
First we run install_driver.bat, it will create file "C:\Program Files (x86)\QualiSystems\CloudShell\Server\Drivers\netscout-teststream.exe" which has to be unblocked.